src/modules/attendees/attendees.service.ts

```diff
@@ -155,6 +195,11 @@ export class AttendeesService {
                 if (deletedAttendees.affected != 1) {
                     throw new BadRequestException('Not deleted');
                 }
+                this.logger.error(
+                    '/Delete attendees details by id',
+                    'Event Attendees for user and eventId deleted successfully.',
```

Comment:
Replace your comment with this msg 
`Event attendee deleted successfully for this userid: [userid] and eventid:[eventid]`